src/ScheduleFunctions.cpp

```diff
@@ -169,20 +182,17 @@ Stmt build_provide_loop_nest_helper(string func_name,
         // Use 'var', the variable which bounds we're constraining as the
         // container name, so that we can use it later to check if a LetStmt
         // value depends on 'var'.
-        Container c1 = {Container::IfInner, 0, dim_var, likely(var >= min)};
-        Container c2 = {Container::IfInner, 0, dim_var, likely(var <= max)};
-        nest.push_back(c1);
-        nest.push_back(c2);
+        nest.emplace_back(Container::IfInner, 0, dim_var, likely(var >= min));
+        nest.emplace_back(Container::IfInner, 0, dim_var, likely(var <= max));
         n_predicates_inner += 2;
     }
 
     // Put all the reduction domain predicates into the containers vector.
     for (Expr pred : predicates) {
         pred = qualify(prefix, pred);
-        Container c = {Container::If, 0, "", likely(pred)};
-        pred_container.push_back(c);
+        pred_container.emplace_back(Container::If, 0, "", likely(pred));
     }
-    int n_predicates = pred_container.size();
+    int n_predicates = static_cast<int>(pred_container.size());
```

Comment:
We use c-style casts for this. Merits of that policy can be debated, but this PR should be consistent with the rest of the compiler.

Reply:
Okay - I'll follow the rest of the project's conventions.